DNN Platform/Library/Entities/Tabs/ITabController.cs

```diff
@@ -246,6 +246,16 @@ public interface ITabController
         /// <returns>tab collection</returns>
         TabCollection GetTabsByPortal(int portalId);
 
+
+        /// <summary>
+        /// Get the actual visible tabs for a given portal id. 
+        /// System Tabs and Admin Tabs are excluded from the result set.
+        /// </summary>
+        /// <param name="portalId"></param>
+        /// <param name="adminTabId"></param>
+        /// <returns>tab collection</returns>
+        TabCollection GetTabsByPortal(int portalId, int? adminTabId);
```

Comment:
I find this API a bit misleading:

* The second parameter is there because `AdminTabId` from portal to portal. This is understandable but this method can be abused by passing any number to produce a weird result
* other than in the comment, the method hides the fact that system pages are also excluded

Here are some ideas:

* a more specific name would be more helpful like `GetUserTabsByPortal` or `GetNonSystemTabs`. 
  * I don't necessarily buy into the `*ByWhatever` suffixes but I'm not adamant
* I would remove the parameter and have the `AdminTabId` retrieved inside. `TabCollection GetUserTabsByPortal(int portalId)` or even `TabCollection GetUserTabsByPortal(PortalInfo portal)` would do.   
  * you would then just call it as `GetUserTabsByPortal(this)`
* **OR** just do the filtering inside the `PortalInfo#Pages`? 🤔

Reply:
GetUserTabsByPortal is the one I prefer 🤔 
Reasons: 

- AdminTabId is something detected by on check for "Admin" string in the page name (some improvement here in 9.2.1?)
- Get**UserPage**ByPortal let us undertand we are not including admin and system pages